service/src/main/resources/application.yml

```diff
@@ -28,6 +28,10 @@ env:
     terra-datarepo: ${TERRA_DATAREPO_URL:https://jade.datarepo-dev.broadinstitute.org}
     bpm: ${BPM_ADDRESS:https://bpm.dsde-dev.broadinstitute.org/}
     tps: ${TPS_ADDRESS:https://tps.dsde-dev.broadinstitute.org/}
+  azure:
+    customer:
+      usage-attribute: ${AZURE_CUSTOMER_USAGE_ATTRIBUTE:}
```

Comment:
If no env. variable found value would be empty string. In this case no header would be set.